### PR TITLE
add benchmark for automation-price

### PR DESCRIPTION
**Pull request by v9n**
Starting adding properly benchmark to prepare for weight and fee handler.

Part of #434

### CODE REVIEW

File: pallets/automation-price/src/lib.rs [hunk 1]
```diff
@@ -314,12 +324,10 @@ pub mod pallet {
 
 	#[pallet::error]
 	pub enum Error<T> {
-		/// The provided_id cannot be empty
 		EmptyProvidedId,
```
Comment (chrisli30) [inline pallets/automation-price/src/lib.rs:327]: This Error related to the providedId is the old clientId pre-2.0.0. Is it being used by automationPrice?

File: runtime/neumann/src/lib.rs [hunk 2]
```diff
@@ -929,6 +929,11 @@ impl pallet_automation_price::Config for Runtime {
 	type UniversalLocation = UniversalLocation;
 	type SelfParaId = parachain_info::Pallet<Runtime>;
 	type EnsureProxy = AutomationEnsureProxy;
+
+	// how many oracle can be authorized to update price for a pair
+	type MaxAuthorizedOracleWallet = ConstU32<5>;
+	// How many item we can put into the same price update extrinsic
+	type MaxBatchPriceUpdate = ConstU32<100>;
```
Comment (chrisli30) [inline runtime/neumann/src/lib.rs:936]: These are good ideas. The variable names are self-explanatory. Since they are new, we will need to document them along with other new events or chain configs on docs.oak.tech. 

I suggest us to keep it simple for now since we will be using a single oracle for the next 3-6 months, because these two parameters will need a set of unit tests to ensure the edge cases or errors are handled correctly

**Review by chrisli30 — APPROVED** [review]
No major issues identified, and I love the code comments 👍
